google-cloud-sdk/PKGBUILD

```
@@ -22,19 +22,15 @@ optdepends=(
 )
 options=('!strip' 'staticlibs')
 source=(
-  "https://dl.google.com/dl/cloudsdk/release/downloads/for_packagers/linux/${pkgname}_${pkgver}.orig.tar.gz"
+  "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/${pkgname}-${pkgver}-linux-${arch}.tar.gz"
```

```suggestion
  "https://dl.google.com/dl/cloudsdk/release/downloads/for_packagers/linux/google-cloud-cli_${pkgver}.orig.tar.gz"
```

Releases that are not from `for_packagers` don't include the man files. The shasum for that file is `1ebb32cc5a83fd9eded10649e579f97c699a9412906fb45688f3f5a07ec3bde3`.

That way you can also revert some of the changes in the `prepare` function.